### ISSUE TITLE
Incompatibiity with Huion Kanvas GT 220 or buggy with Mac

### ISSUE THREAD

**Issue by artbynyaa** [tahoma2d/tahoma2d#636]
**Describe the bug**
When I open the program and try to draw, the cursor is way off of where I'm currently drawing. I re-calibrated my tablet and it still pulled the same issue.

**To Reproduce**
List steps to reproduce the behavior.
1.Open the program on a mac computer.
2.connect your Huion Kanvas GT 220 and try to draw.
3.

**Device Information:**
 - OS: IOS
 - Tahoma2D Version: 1.1
 - GPU:AMD Radeon Pro 5300M 4 GB

**Comment by artisteacher** [comment]
There's definitely an offset issue when drawing in a dual monitor sort of setup if the screen resolutions are different. Making sure the resolutions are the same, mirroring instead of using an extended desktop, and/or switching which is the primary monitor have all been reported as workarounds.

I ran into something similar when Tahoma was already open and I connected a projector that is set to mirror. The resolution switches to be optimized for the projector whenever the projector connects. In my case, if I open Tahoma after the projector is connected, there is no offset when I draw and everything displays properly.

If any of these strategies help, please let us know what worked.

**Comment by artbynyaa** [comment]
Thank you the mirroring worked and fixed it. So it's good to know that it
doesn't work on extended displays currently. Thanks again!

**Comment by DarrenTAnims** [comment]
It can work on extended displays, but it doesn't for everyone. Having the same resolution helps. On my system I have a different resolution, so I'm lucky that it's working ok. I use a Huion Kamvas too.

**Comment by artbynyaa** [comment]
Yeah, I have a mac so I'm using it there on extended display and clip never
had this problem.

**Comment by artisteacher** [comment]
Extended desktop works just fine with Tahoma and an iPad + Sidecar.

**Comment by artbynyaa** [comment]
I wish i had that but i don't lol I have my computer and Huion Kanvas gt
220 which is a display tablet
